src/run.js

```diff
@@ -250,10 +253,6 @@ export default stateType => ({
     } else {
       log('browser change: browser initiated');
       state.next(transitionReducer(mergeStateAndBrowserState(state.value, fromRouter)));
-      // push a dummy transition
-      // so that `transitionReducer` can run
-      // will be diffed away if there's nothing else to be done
-      transition({ ___k: Math.random() });
```

Comment:
this was now:
- unneeded (`transitionReducer` is run a few lines above)
- potentially dangerous on app boot (if routerDiff fails basically you can't use the app for the first `timeoutMSec` milliseconds)